core/components/atoms/avatar/icon/Icon.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Icon } from '@/index';
```

Comment:
Image component have a index file but Icon component have no index file, please use same structures, if directory is not required move component to top level with component name as file name.

Reply:
I missed in structure, Icon also has an index file.